site/_includes/partials/post-banner.njk

```diff
@@ -0,0 +1,9 @@
+<div class="post-banner">
+  <div>
+    {% if new_available_content_url %}
+      {% set material_url = helpers.findByUrl(collections.all, new_available_content_url, locale) %}
+    {% endif %}
+
+    This content is outdated and no longer accurate. {% if material_url %}<a href={{ material_url.url }}>New material available</a>.{% endif %}
```

Comment:
Sorry for not spotting this earlier - this strings need to be passed through i18n filter to make them translatable.